NEWS.md

```diff
@@ -1,5 +1,9 @@
 # admiral (development version)
 
+## Documentation
+
+- Documentation for `derive_extreme_event()` has been updated to include a description for the value of `derive_extreme_event()` when `keep_source_vars = NULL`
```

Comment:
Hi @ProfessorP-beep - can you move this into the already existing Documentation section and place the issue number at the end.  pkgdown makes it into a dynamic link that can take the user to github!